testing/test_case.go

```diff
@@ -17,10 +18,18 @@ func (receiver *TestCase) Seed(seeds ...seeder.Seeder) {
 			command += fmt.Sprintf(" %s", seed.Signature())
 		}
 	}
+	if artisanFacade == nil {
```

Comment:
Judge `artisanFacade == nil` when initing it instead of judging it in every method of this file.